pyproject.toml

```diff
@@ -11,11 +11,10 @@ case-converter = "1.1.0"
 click = "8.0.4"
 click-default-group = "1.2.2"
 setuptools = "60.9.3"
-splight-lib = "4.4.0"
-py7zr = "0.18.5"
+splight-lib = "4.5.4"
```

Comment:
I wouldn't update this dependency, there may be some changes in the code for using this version of the library